Description: Correct hfi1_control's detection of pci.ids
 hfi1_control uses the package manager to find the location of pci.ids.
 However, as shipped, it uses looks in hwdata instead of pciutils.
Author: Brian T. Smith <bsmith@systemfabricworks.com>
Forwarded: no
Last-Update: <2017-12-14>
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/scripts-enduser/hfi1_control
+++ b/scripts-enduser/hfi1_control
@@ -1,4 +1,4 @@
-#! /bin/bash
+#!/bin/bash
 # This file is provided under a dual BSD/GPLv2 license.  When using or
 # redistributing this file, you may do so under either license.
 #
@@ -523,11 +523,11 @@
 
 	# find path to pci.ids by querying hwdata package information
 	if [ "$(cat /etc/issue | egrep -i 'ubuntu|debian')" ]; then
-	    cmd="dpkg-query -L"
+	    cmd="dpkg-query -L pciutils"
 	else
-	    cmd="rpm -ql"
+	    cmd="rpm -ql hwdata"
 	fi
-	pci_ids=$(${cmd} hwdata | grep pci.ids)
+	pci_ids=$(${cmd} | grep pci.ids)
 	# if path was not found in package, use default path
 	if [ -z ${pci_ids} ]; then
 	    pci_ids=${default_pci_ids}
